docs/admin/about-docs.md

```diff
@@ -59,3 +59,24 @@ cd docs
 make html
 ```
 
+## Documentation Markup Keys
+
```

Comment:
Add an explanation of why this is useful. Consider including a link to the index page for the Apps section